package.json

```diff
@@ -65,7 +65,7 @@
     "choo": "^3.2.0",
     "collect-stream": "^1.1.1",
     "dat-design": "^1.4.2",
-    "dat-encoding": "^3.0.0",
+    "dat-key-as": "^1.0.0",
```

Comment:
dat-key-as is a different interface cc @joehand 

i'll keep dat-key-as in this module because its easier for my brain to understand but i hear you on maybe there are too many modules doing the same thing